Kickstarter-iOS/Views/Controllers/ManagePledgeViewControllerTests.swift

```diff
@@ -41,11 +46,42 @@ final class ManagePledgeViewControllerTests: TestCase {
     }
   }
 
+  func testView_CurrentUser_IsNotBacker() {
```

Comment:
I think the name can cause confusion in the future, since only creators and backers will have access to this screen (and for now, we're not showing this screen to creators). So I think we could either delete it or maybe rename and update it to test view when user is creator of project?

Reply:
I think it's helpful to have this screenshot as a reference for how the view should look if the current user is not the backer 🤷‍♀